### DISCUSSION TITLE
欢迎用我的推荐码：XHWFR3能赠送一个月会员。谢谢！

### DISCUSSION THREAD

**Discussion by x860** [getlantern/lantern-forum-cn#932] [Ideas]
欢迎用我的推荐码：XHWFR3能赠送一个月会员。谢谢！

**Comment by ycjdixon** [comment]
送1年也没有用，反正用不了的。浪费钱。

**Reply by MaxMKMax** [reply]
@ycjdixon 您好. 我们是因为更换系统了所以在也支持不了window 7 了. 非常抱歉啊.

**Reply by LuLu120325** [reply]
请帮我处理退款，给个明确的回复，谢谢，邮箱：2647611833@qq.com

**Reply by ycjdixon** [reply]
@Mike777Khoo  WIN11也一样哦
